### PR TITLE
Fix push strings for message timer update events

### PR DESCRIPTION
**Pull request by daehn**
## What's new in this PR?

* Fix push strings for message timer update events.

### CODE REVIEW

File: Source/Notifications/Push notifications/Notification Types/LocalNotificationType+Localization.swift [hunk 1]
```diff
@@ -45,6 +45,7 @@ private let ZMPushStringFailedToSend        = "failed.message"       // "Unable
 private let ZMPushStringMemberJoin          = "member.join"          // "[senderName] added you"
 private let ZMPushStringMemberLeave         = "member.leave"         // "[senderName] removed you"
 private let ZMPushStringMessageTimerUpdate  = "message-timer.update" // "[senderName] set the timed messages to [duration]
+private let ZMPushStringMessageTimerOff     = "message-timer.off"    // "[senderName] set the timed messages to [duration]
```
Comment (mikeger) [inline Source/Notifications/Push notifications/Notification Types/LocalNotificationType+Localization.swift]: Should it be `set the timed messages to Off` ?

Comment (daehn) [reply Source/Notifications/Push notifications/Notification Types/LocalNotificationType+Localization.swift]: yeah the comment is wrong

**Comment by daehn** [comment]
@mikeger updated!

**Comment by daehn** [comment]
@zenkins release patch

**Pull request merged**